components/config/src/config/search.rs

```diff
@@ -1,5 +1,19 @@
 use serde::{Deserialize, Serialize};
 
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+pub enum IndexFormat {
+    #[serde(rename = "json")]
+    Json,
+    #[serde(rename = "javascript")]
+    Javascript,
```

Comment:
We should likely prefix them with elasticlunr as it would the format for those and we want to avoid breaking changes later if we add more format.